Constructive Code Review/outline.txt

```diff
@@ -0,0 +1,205 @@
+- Splash screen
+    - PR screenshot: “Your code is bad and you are bad. Have a bad day.”
+- Importance
+    - Who's done code review before?
+        - Who's done it sync?
+            - We don't really do that, aside from maybe some pair
+              programming.
+        - Async?
+    - Pretty much all code at Mozilla gets reviewed before it gets
+      merged.
```

Comment:
I would say `s/Pretty much all/All/` unless there are exceptions I am unaware of? We even review things like config changes and translations etc.